tests/foreman/api/test_remoteexecution.py

```diff
@@ -24,15 +24,14 @@
 from robottelo.api.utils import wait_for_tasks
 from robottelo.config import settings
 from robottelo.helpers import add_remote_execution_ssh_key
-from robottelo.vm_capsule import CapsuleVirtualMachine
+from robottelo.hosts import Satellite
 
 
 CAPSULE_TARGET_VERSION = '6.9.z'
```

Comment:
why it is hard-coded ? not part of this PR change

Reply:
Used before, we could change to `CAPSULE_TARGET_VERSION = f'{settings.satellite_version}/z'`.